src/Apps/Legion/Components/WizardShell/Internal/ModifyStep/Internal/TypesTab/TypesTab.styles.ts

```diff
@@ -0,0 +1,21 @@
+import { ITypesTabStyles } from './TypesTab.types';
+import {
+    IProcessedStyleSet,
+    mergeStyleSets,
+    memoizeFunction
+} from '@fluentui/react';
+
+export const getStyles = memoizeFunction(
+    (): IProcessedStyleSet<ITypesTabStyles> => {
+        return mergeStyleSets({
+            root: {
+                maxHeight: 400,
```

Comment:
I don't love that teh tab has to know how big the modal is going to be. 🤔 I'm assuming this is what causes scrolling to work right?

Reply:
Will defer this to a later PR for a better solution.